light_node/src/main.rs

```diff
@@ -284,6 +284,9 @@ fn block_on_actors(
         drop(tezos_readonly_api);
         debug!(log, "Shutdown tezos_readonly_api complete");
 
+        persistent_storage.flush_dbs();
+        info!(log, "Flushing databases");
```

Comment:
@michal-stanek 
I think this log should go before `persistent_storage.flush_dbs();` :)